src/Features/AnalyticsDashboard.php

```diff
@@ -46,6 +46,10 @@ public function __construct() {
 		// priority is 20 to run after https://github.com/woocommerce/woocommerce/blob/a55ae325306fc2179149ba9b97e66f32f84fdd9c/includes/admin/class-wc-admin-menus.php#L165.
 		add_action( 'admin_head', array( $this, 'update_link_structure' ), 20 );
 		add_filter( 'woocommerce_admin_preload_options', array( $this, 'preload_options' ) );
+
+		if ( Loader::is_feature_enabled( 'homescreen' ) ) {
+			add_filter( 'woocommerce_admin_plugins_whitelist', array( $this, 'get_homescreen_allowed_plugins' ), 10, 2 );
```

Comment:
I noticed that the method `get_homescreen_allowed_plugins` receives one arg instead of two. Can we remove this `2`? By default, the `accepted_args` is 1.

Reply:
Whoops! Copy 🍝 .